sysutils/pfSense-pkg-LCDproc/files/usr/local/www/packages/lcdproc/lcdproc_screens.php

```diff
@@ -273,6 +275,23 @@
 )->setHelp('A 4&hyphen;row 20&hyphen;column display size, or higher, is recommended for this screen.');
 
 
+$section->addInput(
+	new Form_Checkbox(
+		'scr_interfaces_link', // checkbox name (id)
+		'Interfaces link status', // checkbox label
+		'Display the interfaces current link status', // checkbox text
+		$pconfig['scr_interfaces_link'] // checkbox initial value
+	)
+)->setHelp(
+	'This will create a seperate status screen for each inferface with four lines each:<br />' .
+	'&lt;IFNAME&gt;: &lt;link status&gt;<br />' .
+	'v4: &lt;ip v4 address&gt;<br />' .
+	'v6: &lt;ip v6 address&gt;<br />' .
+	'm: &lt;mac address&gt;<br />' .
+	'A 4&hyphen;row 20&hyphen;column display size, or higher, is recommended for this screen.'
```

Comment:
Avoid adding html entities or tags inside the Help message. This string is considered to be added to translation files and it's easy to be broken if a translator make a mistake with them. `setHelp()` can be used as `printf()` so you can use parameters like `%1$s` as used in other places in pfSense main code

Reply:
Do you mean like this?
```
)->setHelp(
	'This will create a seperate status screen for each inferface with four lines each:%1$s' .
	'%2$sIFNAME%3$s: %2$slink status%3$s%1$s' .
	'v4: %2$sip v4 address%3$s%1$s' .
	'v6: %2$sip v6 address%3$s%1$s' .
	'm: %2$smac address%3$s%1$s' .
	'A 4%4$srow 20%4$scolumn display size, or higher, is recommended for this screen.',
	['<br/>', '&lt;', '&gt;', '&hyphen;', ]
);
```

Reply:
Exactly! Can you please make the change?